src/Nox.Types.EntityFramework/Types/Yaml/YamlConverter.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework.Types;
+
+/// <summary>
+/// Converts a <see cref="Yaml"/> to and from a <see cref="string"/> for Entity Framework.
+/// </summary>
+public class YamlConverter: ValueObjectConverter<Yaml, string>
```

Comment:
we will remove the ValueObjectConverter since its creating an instance to do the FromDatabase

@gjorgji-grgovski-authoritypartners I'm approving but then we need to take car of this converter also